drf_spectacular/views.py

```diff
@@ -80,11 +82,25 @@ class SpectacularSwaggerView(APIView):
 
     @extend_schema(exclude=True)
     def get(self, request, *args, **kwargs):
+        static_absolute_path = "//unpkg.com/swagger-ui-dist"
+        swagger_bundle_js_href = "{}@{}/swagger-ui-bundle.js".format(static_absolute_path, spectacular_settings.SWAGGER_UI_DIST_VERSION)
+        swagger_bundle_css_href = "{}@{}/swagger-ui.css".format(static_absolute_path, spectacular_settings.SWAGGER_UI_DIST_VERSION)
+        swagger_bundle_favicon_href = "{}@{}/favicon-32x32.png".format(static_absolute_path, spectacular_settings.SWAGGER_UI_DIST_VERSION)
+
         schema_url = self.url or reverse(self.url_name, request=request)
+        swagger_ui_config = spectacular_settings.SWAGGER_UI_CONFIG
+        if swagger_ui_config.get('url') is None:
```

Comment:
with the other #163 solution this is not necessary anymore. it supports overriding.